### DISCUSSION TITLE
What happened to the SDK's internal layout in version 2.0.8+? The sdk zip on Maven appears to be a "delta" download.

### DISCUSSION THREAD

**Discussion by CalculonPrime** [GoogleCloudPlatform/appengine-java-standard#320] [Q&A] [answered]
The top level directory in the SDK is unexpected by our toolchain and is named:
**google_appengine_java_delta**
This started in SDK version 2.0.8.
So it's a delta download?  Can you please return to the standard asset layout in the SDK zip?

**Comment by ludoch** [comment]
Hi, can you provide more details about the files you are looking at?
Is it https://repo1.maven.org/maven2/com/google/appengine/appengine-java-sdk/2.0.31/ ?
Or different artifacts?

**Reply by CalculonPrime** [reply]
Yes, you can see it there if you download appengine-java-sdk-2.0.31.zip, for example.

The top level directory is google_appengine_java_delta instead of being the various expected subdirectories.  For example, in 2.0.7, the last version before this change, lib/appengine-tools-api.jar was the path to that jar. That's no longer a valid path.

**Reply by maigovannon** [reply, marked as answer]
Hi I am personally am not entirely sure about the the timeline of this change. However, I just check the tarball version of this and it should have the format you are expecting: https://repo1.maven.org/maven2/com/google/appengine/appengine-java-sdk/2.0.31/appengine-java-sdk-2.0.31.tar.gz.

Could you please download this and see if this works? Meanwhile, I will check with @ludoch and get back on the ZIP changes

**Reply by CalculonPrime** [reply]
OK, yeah, apparently the tar.gz file is in the expected layout.  Only the zip file has the weird structure.  We chose the zip format because we grab the GWT SDK in zip format and thought it would make sense to do the same for the App Engine.
